Makefile

```diff
@@ -57,8 +57,10 @@ test:
 
 .PHONY: cover
 cover:
-	go test -race -coverprofile=cover.out -coverpkg=./... ./...
-	go tool cover -html=cover.out -o cover.html
+	@$(foreach dir,$(MODULE_DIRS), ( \
```

Comment:
See line 10 in this file (I miss phab already) - looks like we were ignoring exp on purpose (?)

We should perhaps at least change the comment?

Reply:
oops, thanks for catching that. Actually it looks like that comment predates adding exp package. I'll remove that comment and change this up a bit